components/playlist/media_detector_component_manager.cc

```diff
@@ -138,12 +138,18 @@ void MediaDetectorComponentManager::SetUseLocalScriptForTesting() {
       }
   }
 
-  function getNodeSource(node, src, mimeType, thumbnail) {
+  function getNodeData(node) {
+    var src = node.src
+    var mimeType = node.type
     var name = node.title;
     if (name == null || typeof name == 'undefined' || name == "") {
-      name = document.title;
+      // Try getting mobile youtube specific data
+      name = window.ytplayer?.bootstrapPlayerResponse?.videoDetails?.title
     }
 
+    if (name == null || typeof name == 'undefined' || name == "")
+      name = document.title
```

Comment:
let's try to be consistent here with semicolons since the JS linter is not running on these files

Reply:
> JS linter is not running on these files

Yeah, that's a problem. I guess I can split this into a separate JS file and run eslint for consistency.